Source/WebCore/accessibility/AXObjectCache.h

```diff
@@ -166,6 +166,8 @@ class AccessibilityReplacedText {
     macro(ColumnIndexChanged) \
     macro(ColumnSpanChanged) \
     macro(ContentEditableAttributeChanged) \
+    macro(CommandChanged) \
+    macro(CommandForChanged) \
```

Comment:
Here too we like to sort:
```suggestion
    macro(CommandChanged) \
    macro(CommandForChanged) \
    macro(ContentEditableAttributeChanged) \
```